src/index.html

```diff
@@ -5,9 +5,63 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card-image">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27-image"
+          class="card-image__model"
+        >
+
+        <h2 class="card-image__info">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="card-image__code">
+          Product code: 195434
+        </h3>
+
+        <div class="star stars--4">
+          <div class="star__stars"></div>
+          <div class="star__stars"></div>
+          <div class="star__stars"></div>
+          <div class="star__stars"></div>
+          <div class="star__stars"></div>
+
+          <div>
+            <div class="star__reviews">
+              Reviews: 5
+            </div>
+          </div>
+        </div>
+
+        <div class="card-image__text">
+          Price:
+          <div class="card-image__cost">
+            $2,199
+          </div>
+        </div>
+
+        <div class="button">
+          <div>
+            <a href="#" data-qa="hover">
+              <button class="button-buy">
+                Buy
+              </button>
+            </a>
+          </div>
+        </div>
```

Comment:
you don't need so many wrappers
actually, there might be only `a` with correct styles, smth like that:
```suggestion
        <a href="#" data-qa="hover" class="button">
          Buy
        </a>
```